js/modules/k6/http/http.go

```diff
@@ -37,36 +38,101 @@ import (
 	"github.com/dop251/goja"
 	"github.com/loadimpact/k6/js/common"
 	"github.com/loadimpact/k6/js/modules/k6/html"
+	"github.com/loadimpact/k6/lib"
 	"github.com/loadimpact/k6/lib/netext"
 	"github.com/loadimpact/k6/stats"
 	"github.com/pkg/errors"
+	"golang.org/x/crypto/ocsp"
 )
 
 var (
 	typeString = reflect.TypeOf("")
 	typeURLTag = reflect.TypeOf(URLTag{})
 )
 
+type OCSP struct {
+	ProducedAt, ThisUpdate, NextUpdate, RevokedAt int64
+	RevocationReason                              string
+	Status                                        string
+}
+
 type HTTPResponseTimings struct {
 	Duration, Blocked, LookingUp, Connecting, Sending, Waiting, Receiving float64
 }
 
 type HTTPResponse struct {
 	ctx context.Context
 
-	RemoteIP   string
-	RemotePort int
-	URL        string
-	Status     int
-	Proto      string
-	Headers    map[string]string
-	Body       string
-	Timings    HTTPResponseTimings
-	Error      string
+	RemoteIP       string
+	RemotePort     int
+	URL            string
+	Status         int
+	Proto          string
+	Headers        map[string]string
+	Body           string
+	Timings        HTTPResponseTimings
+	TLSVersion     string
+	TLSCipherSuite string
+	OCSP           OCSP `js:"ocsp"`
+	Error          string
 
 	cachedJSON goja.Value
 }
 
+func (res *HTTPResponse) setTLSInfo(h *HTTP, tlsState *tls.ConnectionState) {
+	switch tlsState.Version {
+	case tls.VersionSSL30:
+		res.TLSVersion = h.SSL_3_0
```

Comment:
These constants should honestly be global `const`s, passing in the HTTP object just to read them feels silly.